src/components/VideoGallery.tsx

```
@@ -0,0 +1,343 @@
+    return true;
+  });
+
+  // Sort videos based on selected sort option
+  const sortedVideos = [...filteredVideos].sort((a, b) => {
+    switch (sortBy) {
+      case "newest":
+        return b._creationTime - a._creationTime;
+      case "oldest":
+        return a._creationTime - b._creationTime;
+      case "duration":
+        const durationA = parseInt(a.duration) || 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)